tests/test_onehot_encoder.py

```diff
@@ -0,0 +1,49 @@
+# Load Libraries
+from encoderPy import target_encoder
+import pandas as pd
+import numpy as np
+
+# Load depression test data
+data = pd.read_csv("https://raw.githubusercontent.com/UBC-MDS/encoderPy/master/data/depression_data.csv")
```

Comment:
I think it would be better to input the local file, so that users won't have to connect to the internet after they install the package.
```suggestion
data = pd.read_csv("data/depression_data.csv")
```